pkg/reconciler/common/transformers.go

```diff
@@ -32,3 +33,27 @@ func Transformers(ctx context.Context, obj v1alpha1.KComponent) []mf.Transformer
 		ConfigMapTransform(obj.GetSpec().GetConfig(), logger),
 		ResourceRequirementsTransform(obj.GetSpec().GetResources(), logger)}
 }
+
+// Transform will replace the passed-by-reference manifest with one
+// transformed by platform, common, and any extra passed in
+// TODO: deal with platforms better, i.e. encapsulate their clientset
+func Transform(ctx context.Context, manifest *mf.Manifest, instance v1alpha1.KComponent, client kubernetes.Interface, extra ...mf.Transformer) error {
+	logger := logging.FromContext(ctx)
+	logger.Debug("Transforming manifest")
+	platforms := GetPlatforms(ctx)
```

Comment:
Is this really present on the "reconciling" context? Where is it added?

Reply:
Apparently not. :) Restored, thanks.